packages/devextreme-react-generator/src/component-generator.ts

```diff
@@ -549,6 +564,16 @@ function generate(component: IComponent): string {
       .map((t) => renderPropTyping(createPropTypingModel(t)))
     : undefined;
 
+  const hasExplicitTypes = !!component.optionsTypeParams?.length;
+
+  const filterReexports = (reexports?: string[]) : string[] => (
+    reexports
+      ? reexports.filter(
+        (item) => item !== 'default' && (hasExplicitTypes && item !== 'ExplicitTypes'),
```

Comment:
Can this be simplified?
```suggestion
        (item) => item !== 'default' && tem !== 'ExplicitTypes',
```

Reply:
fixed